src/components/formInput/FormInput.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from "./formInput.module.css"
+
+export const FormInput = ({
+	label,
+	inputType = "text",
+	placeholder,
+	onChange,
+	isRequired = false,
+}: {
+	label: string
+	inputType?: string
+	placeholder: string
+	onChange: (e: any) => void
+	isRequired?: boolean
+}) => {
+	return (
+		<div className={styles.inputWrapper}>
+			<label className={styles.label} htmlFor={label}>
```

Comment:
This `htmlFor` should ideally take an id value, which corresponds to the input element